pkg/funcs/functions_test.go

```diff
@@ -0,0 +1,92 @@
+package pkg
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/cloudprivacylabs/lpg"
+	"github.com/cloudprivacylabs/lsa/pkg/ls"
+	"github.com/cloudprivacylabs/opencypher"
+)
+
+type ageTest struct {
+	query    string
+	expected int
+}
+
+func TestAge(t *testing.T) {
+	rGetter := func(v opencypher.Value) interface{} {
+		return v.Get().(opencypher.ResultSet).Rows[0]["1"].Get()
+	}
+	ageTests := []ageTest{
+		{
+			query:    `return age(parseDate("01/01/2000", "MM/DD/YYYY"))`,
```

Comment:
This test will not age well. It will start failing in January. Add a GetNow() func variable to the functions.go,initialize it to time.Now, change it for the test.